molfeat/trans/pretrained/hf_transformers.py

```diff
@@ -397,7 +398,10 @@ def _embed(self, inputs, **kwargs):
         else:
             attention_mask = None
         with torch.no_grad():
-            out_dict = self.featurizer.model(output_hidden_states=True, **inputs)
+            if hasattr(self.featurizer.model, 'encoder'):
```

Comment:
Use the `use_encoder` if the model is a huggingface `EncoderDecoder` model to check whether `encoder` should be used. Otherwise default to what was done before.

Reply:
I forgot to delete the use_encoder, but I figured the hasattr() might be better because it allows you to avoid adding another init parameter. Is using init better in this scenario?

Reply:
It's fine the way you proposed if you delete the `use_encoder`, I wanted to make it more general to support most cases of `EncoderDecoder` models.